routes/blog.js

```diff
@@ -1,55 +1,13 @@
-const articleModel = require('../models/blog')
 const express = require('express')
 const app = module.exports = express()
-const bodyParser = require('body-parser')
+const blog = require('../controllers/blog')
 
+app.get('/blog', blog.getAllArticles)
 
-// Read for the blog CRUD operation 
-//on the articles page list all article
-app.get('/blog', (req, res) => {
-  articleModel.find()
-    .then((result) => {
-      res.send(result)
-    })
-    .catch((err) => {
-      console.log(err)
-    })
-})
+app.get('/blog/:articleId', blog.getSingleArticle)
 
-//Delete  for the CRUD operation
-//delete the article by it's id
-app.delete('/blog/delete/:articleId', (req, res) => {
-  articleModel.findByIdAndDelete(req.params.articleId)
-    .then((result) => {
-      if (!result) {
-        res.send(`article with id ${req.params.articleId} can't be found`)
-      }
-      else {
-        res.send('article deleted successfully')
-      }
-    })
-})
+app.post('/blog/add', blog.createNewArticle)
```

Comment:
resources in REST-API should be in plural. eg blogs.
And your URLs can be like:
 GET - https://localhost:5000/api/v1/blogs 
 POST - https://localhost:5000/api/v1/blogs 
 etc

Reply:
okay